chunky/src/java/se/llbit/chunky/block/MinecraftBlock.java

```diff
@@ -7,7 +7,12 @@
  * A simple opaque block with a single texture.
  */
 public class MinecraftBlock extends Block {
-  public static final Material STONE = new MinecraftBlock("stone", Texture.stone);
+  public static final Material STONE = new MinecraftBlock("stone", Texture.stone) {
+    @Override
+    public Block applyWaterlogging() {
+      throw new UnsupportedOperationException();
```

Comment:
I wonder if this should only throw in a dev environment, just to avoid crashing in prod

Though chunky in general probably needs this everywhere

Reply:
I was wondering too, but this particular `STONE` instance is only used in the preview renderer. I'll replace this with a `return this` instead. Waterlogged stone would just be stone anways. :shrug: